src/ucp/wireup/wireup_ep.c

```diff
@@ -470,10 +470,7 @@ ssize_t ucp_wireup_ep_sockaddr_fill_private_data(void *arg, const char *dev_name
     ucp_worker_iface_t *wiface;
     ucs_status_t status;
     uint64_t tl_bitmap;
-    unsigned sockaddr_aux_tls_str_len =
-                    ucs_count_one_bits(context->config.sockaddr_aux_rscs_bitmap) *
-                                       2 * UCT_TL_NAME_MAX;
-    char *sockaddr_aux_tls = ucs_alloca(sockaddr_aux_tls_str_len);
+    char aux_tls_str[UCS_SOCKADDR_AUX_TLS_STRING_LEN];
```

Comment:
just do char aux_tls_str[64]
and then sizeof(aux_tls_str) instead of UCS_SOCKADDR_AUX_TLS_STRING_LEN